.golangci.yml

```diff
@@ -7,7 +7,7 @@ run:
 linters:
   disable-all: true
   enable:
-    - deadcode
+#    - deadcode
```

Comment:
Temporarily disabling these linters as we're leaving some dead code intentionally right now to not make the PR too big. We're doing a follow up with some more cleaning after this one.